docs/source/operations/conversions/unitconvert.rst

```diff
@@ -124,12 +132,31 @@ The same list can also be produced on the command line with :program:`proj` or
 | ind-ch   | Indian Chain                    |
 +----------+---------------------------------+
 
+.. _angular_units:
+
+Angular units
+###############################################################################
+
+In the table below all angular units supported by PROJ `unitconvert` are listed.
+
+.. versionadded:: 5.2.0
```

Comment:
Of you put it right below the heading it is more clear that it is the whole section that was added in version 5.2.0

Reply:
fixed